libs/design-system-components/src/components/ExtensionEditStep2Form/NSFW/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
This is duplicate of ds-components/EditProfile/NSFW

Reply:
it's the same functionality, but the control passed from react-hook-form is different here

Reply:
you mean the typings for the control object because the control is passed as a prop

Reply:
yes, the typings, do you think it would make sense to have more generic typings for it and reuse the component?